modules/oe_webtools_media/tests/Behat/WcloudContext.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_media\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Behat context for Wcloud.
```

Comment:
This is all very well, but the module is called `oe_webtools_media_http_mock`...so by the name, we are talking about a generic test module for webtools media, not wcloud specific...

So either we make a specifc wcloud test module just for this (preferably) or we rename the context and tag to be more generic.

Reply:
I moved it separately but I actually think it's not needed anymore since we enable the mock module in the pipeline for ewcms and it's not being used in here.

Reply:
I removed it..